content/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Open Terms Archive documentation
+linkTitle: Homepage
+weight: 1
+---
+
+# Open Terms Archive documentation
+
+Open Terms Archive is a decentralised system that tracks collections of services' terms across multiple servers. Each collection operates its own API, and the Federation API unifies search and discovery across collections, fostering collaboration with external applications.
+
+### Documentation structure
+
+In this documentation you can find the following kind of content to support users at different levels:
+
+- **Tutorials:** Step-by-step learning guides that help beginners get started with Open Terms Archive, providing foundational knowledge and hands-on experience.
+- **How-to guides:** Task-focused instructions that help experienced users accomplish specific goals efficiently and effectively.
+- **Reference:** Comprehensive technical documentation detailing system components, configuration options, and specifications for advanced users.
+
+### Main contents
+
+- **[Analysis](/analysis/):** Provides guidance on how to analyze terms changes, from navigating through the history of tracked documents to publishing memos about significant changes, along with copywriting guidelines for creating analysis content.
+- **[Community](/community/):** Provides information on how to join and participate in the Open Terms Archive community.
+- **[Terms](/terms/):** Provides guidance on tracking and maintaining terms declarations, reference documentation for declaration formats, and guidelines for selecting selectors and reviewing contributions.
+- **[Collections](/collections/):** Provides guidance on creating and managing collections of tracked terms, creating repositories and defining metadata, and detailed reference documentation for configuration options, environment variables, governance roles, and collection metadata formats.
+- **[Federation](/federation/):** Explains the benefits of joining the Open Terms Archive federation, and the criteria that collections must meet to be part of the federated ecosystem.
+- **[Deployment](/deployment/):** Guides to deploy a collection.
+- **[Programmatic access](/api/):** Provides documentation for programmatically interacting with Open Terms Archive.
+- **[Concepts and principles](/concepts/):** Explains the main concepts and terminology of Open Terms Archive and fundamental design principles.
```

Comment:
Links to section indexes don't work if there is no index :)

And prefer to use `[My Link]({{< relref "path/to/page" >}})` with Hugo.